src/mapgen/mapgen.cpp

```diff
@@ -695,15 +692,7 @@ void MapgenBasic::generateBiomes()
 				if (!biome || y < biome_y_min) {
 					// (Re)calculate biome
 					biome = biomegen->getBiomeAtIndex(index, v3s16(x, y, z));
-
-					// Finding the height of the next biome
-					// On first iteration this may loop a couple times after than it should just run once
-					while (y < biome_y_min) {
-						biome_y_min = biome_transitions[++cur_biome_depth];
-					}
-
-					/*if (x == node_min.X && z == node_min.Z)
-						printf("Map: check @ %i -> %s -> again at %i\n", y, biome->name.c_str(), biome_y_min);*/
```

Comment:
This is helpful for debugging (needed it to compare master against this PR). Could we perhaps keep these two lines for future refactors of this code?

Reply:
we can, but it shouldn't be printf